WooCommerce/Classes/POS/ViewHelpers/KeyboardObserver.swift

```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+import Combine
+
+final class KeyboardObserver: ObservableObject {
```

Comment:
Happy to get some feedback on this `KeyboardObserver`. I wasn't quite sure how to handle the dynamic height for the buttons when keyboard is on/off or when we present messages like error/change due. Right now it seems to do the job but I'm unaware if there are better solutions 🤔

Reply:
I don't know... but you might not need any of it. I've certainly never had to worry about it in SwiftUI before.

[This article](https://www.fivestars.blog/articles/swiftui-keyboard/) suggests that we should just be able to rely on SwiftUI to handle it for us. We can wrap it in a scroll view and it might just work...

At larger dynamic type sizes we'll need one anyway; the button gets hidden by the keyboard at the moment.